plugins/inventory/proxmox.py

```diff
@@ -360,7 +362,17 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
                         agent_iface_key = self.to_safe('%s%s' % (key, "_interfaces"))
                         properties[agent_iface_key] = agent_iface_value
 
-                if config not in plaintext_configs and not isinstance(value, int) and all("=" in v for v in value.split(",")):
+                if config == "lxc":
+                    out_val = {}
+                    for key_list in value:
+                        k, v = key_list
+                        if k.startswith('lxc.'):
+                            k = k.replace('lxc.', '')
```

Comment:
It's better to do
```suggestion
                            k = k[len('lxc.'):]
```
or
```suggestion
                            k = k.replace('lxc.', '', 1)
```
so that you only trim the first `lxc.`. Otherwise, something like `lxc.mylxc.foo` is converted to `myfoo`.

Reply:
Integrated.